gateway-js/src/__tests__/integration/configuration.test.ts

```diff
@@ -187,3 +188,52 @@ describe('gateway startup errors', () => {
     `);
   });
 });
+
+describe('gateway config / env behavior', () => {
+  let gateway: ApolloGateway | null = null;
+  afterEach(async () => {
+    if (gateway) {
+      await gateway.stop();
+      gateway = null;
+    }
+  });
+
+  // TODO(trevor:cloudconfig): this behavior will be updated
+  describe('schema config delivery endpoint configuration', () => {
+    it('A code config overrides the env variable', async () => {
+      let cleanUp = mockedEnv({
```

Comment:
Not a serious suggestion for this PR, but `mocked-env` would be nicer with a syntax like

```
withMockedEnv({FOO: 'bar'}, () => {
  // do stuff
});
```

where you didn't have to stress about ensuring that the (global!!!) state gets cleaned up if the test body throws...

That said, since you already have an afterEach going on you might as well move the `let cleanUp` to under `let gateway` and call it (if it's set) in afterEach?

(Not sure why this is let instead of const... did you try what I had above and it didn't work for some reason I'm missing and then you moved it down here?)

Reply:
Not sure why I had it as let before, but the `afterEach` is fine to me.
b2ff29c